File: C/URI/1019.c
#include <stdio.h>
 
int main() {

   int tempo, horas, horas_seg, minutos, segundos;
   horas_seg=3600;
   scanf("%d", &tempo);
   horas = (tempo/horas_seg);
   minutos = (tempo -(horas_seg*horas))/60;
   segundos = (tempo -(horas_seg*horas)-(minutos*60));
   printf("%d:%d:%d\n",horas, minutos, segundos);
   return 0;
}

File: C#/URI/1019.cs
using System; 

class URI {

    static void Main(string[] args)       
        {
            int tempo, horas, horas_seg, minutos, segundos;
            horas_seg = 3600;

            tempo = Convert.ToInt32(Console.ReadLine());


            horas = (tempo / horas_seg);

            minutos = (tempo - (horas_seg * horas)) / 60;

            segundos = (tempo - (horas_seg * horas) - (minutos * 60));


            Console.WriteLine(horas + ":" + minutos + ":" + segundos);
        }

}